backend/src/model/dto.rs

```diff
@@ -218,6 +218,12 @@ pub struct MapDto {
     pub harvested: i16,
     /// The id of the owner of the map.
     pub owner_id: i32,
+    /// A flag indicating if this map is private or not.
+    pub is_private: Option<bool>,
```

Comment:
We have 3 possible privacy settings, so it must be enum, see doc/usecases/current/map_creation.md

Why is it optional?